### PR TITLE
SHOULD *NOT* GO UPSTREAM

### PR DESCRIPTION
**Pull request by dx-pbuckley**
WIP, untested - allow deploying matisse to chewy internal artifactory

### CODE REVIEW

**Comment by dx-pbuckley** [comment]
@zacharyyy mostly FYI as I haven't tried building it yet in bamboo, let me know if anything glaring looks awry

File: publish.gradle [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.4.17'
+    }
+}
+
+publishing {
+  publications {
+    maven(MavenPublication) {
+      groupId 'com.zhihu.android'
+      version = "0.4.4-SNAPSHOT"
+      artifactId 'matisse'
```
Comment (dx-pbuckley) [inline publish.gradle]: @zacharyyy we have full powers to set the groupId, version, and artifactId in some way that makes sense for us, and differentiates that this is the chewy fork, while still retaining the info that this is the open source library (and not some internal thing). Do you have any preferences for how we do that, or would you prefer that I set it to something that I think will work?

Comment (zacharyyy) [reply publish.gradle]: please set it to anything you think will work. we're finally ready to start consuming this.

Comment (zacharyyy) [reply publish.gradle]: groupId "com.chewy.android.matisse"
version 1.0.0 (probably)
artifactId 'matisse'

**Comment by srassChewy** [comment]
Is this for the coding test or do we have live things out there?

**Pull request merged**